modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -55,7 +55,11 @@ export default class Tile2DHeader {
     });
 
     // A tile can be cancelled while being scheduled
-    if (!requestToken || this._isCancelled) {
+    if (this._isCancelled) {
+      requestToken && requestToken.done(); // eslint-disable-line no-unused-expressions
```

Comment:
Move this if statement after `if(!requestToken)` so you won't have to check this